features/dd-sdk-android-rum/src/main/kotlin/com/datadog/android/rum/internal/vitals/JankStatsActivityLifecycleListener.kt

```diff
@@ -160,6 +215,26 @@ internal class JankStatsActivityLifecycleListener(
         activeActivities[window] = list
     }
 
+    private fun registerMetricListener(window: Window, activity: Activity) {
+        if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.S) {
+            val handler = Handler(Looper.getMainLooper())
+            window.addOnFrameMetricsAvailableListener(DDFrameMetricsListener.DEFAULT, handler)
+        } else {
+        // Fallback - Android 30 allows apps to not run at a fixed 60hz, but didn't yet have
+        // Frame Metrics callbacks available
+        if (Build.VERSION.SDK_INT == Build.VERSION_CODES.R) {
+            val displayManager = activity.getSystemService(Context.DISPLAY_SERVICE) as DisplayManager
+            display = displayManager.getDisplay(Display.DEFAULT_DISPLAY)
+        }
+    }
+}
```

Comment:
latest version of `ktlint` we are using have indent formatting turned off (for a while, we have a ticket for it), so it is better to format code with Android Studio to fix the indent.

```suggestion
    private fun registerMetricListener(window: Window, activity: Activity) {
        if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.S) {
            val handler = Handler(Looper.getMainLooper())
            window.addOnFrameMetricsAvailableListener(DDFrameMetricsListener.DEFAULT, handler)
        } else if (Build.VERSION.SDK_INT == Build.VERSION_CODES.R && display == null) {
            // Fallback - Android 30 allows apps to not run at a fixed 60hz, but didn't yet have
            // Frame Metrics callbacks available
            val displayManager = activity.getSystemService(Context.DISPLAY_SERVICE) as DisplayManager
            display = displayManager.getDisplay(Display.DEFAULT_DISPLAY)
        }
    }
```

Also I think we may set `display` only once (if `DisplayManager` doesn't depend on a particular UI context though).